apps/prairielearn/src/tests/sync/assessmentsSync.test.ts

```diff
@@ -2301,7 +2301,7 @@ describe('Assessment syncing', () => {
     console.log(syncedData.assessment.sync_errors);
     assert.match(
       syncedData.assessment?.sync_errors,
-      /The following questions are marked as draft and therefore cannot be used in assessments: __drafts__\/draft_5/,
+      /The following questions are marked as draft and therefore cannot be used in assessments: __drafts__\/draft_1/,
```

Comment:
@nwalters512 I think this fixed the test? I can't see why you'd want to check if draft_5 was erroring if it never existed and you add draft_1. Please let me know if it's not.

Reply:
Good catch, I failed to commit the second half of a change 😅 yes, `draft_1` is what we should be checking for.